src/firefly/js/ui/PopupUtil.jsx

```diff
@@ -28,9 +28,9 @@ export const MODAL_DIALOG_ID = 'ModalDialog';
 export function showModal(content, show=true) {
     if (show) {
         const dialogContent= (
-            <ModalDialog>
+            <ModasholDialog>
                 {content}
-            </ModalDialog>
+            </ModasholDialog>
```

Comment:
Looks like 'sho' was inserted in the name of ModalDialog (lines 31,33) by accident.